relay/ethToElrond/steps/mock/bridgeExecutorMock.go

```diff
@@ -1,12 +1,16 @@
 package mock
 
+import (
+	"runtime"
+	"sync"
+)
+
+var fullPath = "github.com/ElrondNetwork/elrond-eth-bridge/relay/ethToElrond/steps/mock.(*BridgeExecutorMock)."
+
 // BridgeExecutorMock -
 type BridgeExecutorMock struct {
-	NumCalledPrintDebugInfoCalled  int
-	NumCalledGetPendingBatchCalled int
-	NumHasPendingBatchCalled       int
-	NumCalledIsLeaderCalled        int
-	NumCalledProposeTransferCalled int
+	FunctionCalledCounter map[string]int
```

Comment:
maybe make this unexported as to not try to use it without locking the mutex and running into race conditions?

Reply:
Can be done on a separate PR